src/cases/components/role-exclusions/role-exclusions.component.html

```diff
@@ -0,0 +1,41 @@
+<h2 class="govuk-heading-m">Exclusions</h2>
+
+<table class="govuk-table" *ngIf="exclusions && exclusions.length > 0">
```

Comment:
GOV.UK Design System has a Summary List component that might be more usable. Would simplify the code and save having to set the font size on every single cell. GDS https://design-system.service.gov.uk/components/summary-list/

Reply:
I used the given UI template for this. I did not come up with this. Any change I would have to involve Simon.

Reply:
Discussed and reviewed the GDS guidelines and this component follows the design in the prototype.